index.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/fddcccfd38.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Local TubeClone</title>
+</head>
+
+<body>
+    <!--header start-->
+    <header>
+        <nav class="flex-container">
+            <i class="fas fa-bars"></i>
+            <img class="logo" src="images/logo" alt="image of logo">
+            <form action="">
+                <input class="myInput" type="text" id="search" placeholder="Search"><i class="fas fa-search"></i>
+            </form>
+
+            <i class="fas fa-upload"></i>
+            <button class="sign" type="submit">Sign in</button>
+        </nav>
+        <article>
+            <div class="mini-banner">
+                <p><i class="far fa-user"></i></p>
+                <h6>Choose your Language.</h6>
+                <a href="#" class="learn_more">X</a>
+            </div>
+            <div class="language">
+                <p class="smaller smaller-left">You are viewing Youtube in <b>English (US)</b>. You can <a href="#">Change this preference
+                        below.</a></p> <a class="smaller smaller-right" href="#">Learn More</a>
+
+            </div>
+        </article>
+    </header>
+    <!--header ends-->
+
+    <!--main content-->
+    <div class="container">
+        <main>
+            <div class="video-container">
+                <iframe class="vid" src="https://www.youtube.com/embed/KG3phuIo_JU"></iframe>
+            </div>
+            <div class="activity">
+                <div class="description">
+                    <h3 class="feature-video-title">Watch NBC News NOW Live - August 17</h3>
+                    <div class="description-on-left">
+                        <div class="avatar">
+                            <h2 class="big_t">T</h2>
+                        </div>
+                        <div class="avatar-name">
+                            <h4 class="avatar-title">The Odin Project</h4>
+                        </div>
+                        <div>
+                            <button class="subscribe" type="button"><i class="fab fa-youtube"></i>Subscribe</button>
+                            <div class="number_of_subscribers">987</div>
+                        </div>
+                        <div class="views">32,803 </div>
```

Comment:
- [ ] **OPTIONAL** You can include the word "views" here so that it looks like the original